docs/src/main/asciidoc/smallrye-metrics.adoc

```diff
@@ -153,74 +149,87 @@ public class PrimeNumberChecker {
 }
 ----
 
+[id="running-and-using-the-application_{context}"]
 == Running and using the application
 
-To run the microservice in dev mode, use `./mvnw clean compile quarkus:dev`
-
-=== Generate some values for the metrics
-First, ask the endpoint whether some numbers are prime numbers.
+To execute the application created in xref:writing-an-application_{context}[], do the following:
 
+. Run the microservice in the dev mode:
++
 [source,bash]
 ----
-curl localhost:8080/350
+$ ./mvnw clean compile quarkus:dev
```

Comment:
Please change to `./mvnw clean quarkus:dev` 
(`compile` is not necessary anymore, that is a remnant from old Quarkus versions)